Highlight/zh-Hans.lproj/Preferences.strings

```diff
@@ -0,0 +1,74 @@
+
+/* Class = "NSTextFieldCell"; title = "Languages"; ObjectID = "04A-gG-WMa"; */
+"04A-gG-WMa.title" = "语言";
+
+/* Class = "NSMenu"; title = "Languages"; ObjectID = "2PS-QP-pcK"; */
+"2PS-QP-pcK.title" = "语言";
+
+/* Class = "NSButtonCell"; title = "Bug Report"; ObjectID = "3S7-rh-Mnk"; */
+"3S7-rh-Mnk.title" = "报告 Bug";
```

Comment:
I think users of Highlight should be somewhat familiar with the term "Bug". "错误报告" is more like "Report Issue" since "错误" is a general issue or error instead of an explicit bug in software. But yes, we should consider ordinary users that may just start to learn programming and are not familiar with such an expression, so I just replaced it. Google did it really well : )